kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-case-mgmt/src/test/java/org/kie/server/remote/rest/casemgmt/CaseQueryResourceTest.java

```diff
@@ -17,10 +17,10 @@
 package org.kie.server.remote.rest.casemgmt;
```

Comment:
I think it would be good to add a test method here for calling method `getCaseInstances` for calling the underling new overloaded method `getCaseInstancesAnyRole` withData=true